<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0,0">
    <item row="0" column="3">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="5,1">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="pushBack_btn">
          <property name="text">
           <string>Push_back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushFront_btn">
          <property name="text">
           <string>Push_front</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="popBack_btn">
          <property name="text">
           <string>Pop_back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="popFront_btn">
          <property name="text">
           <string>Pop_front</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clear_btn">
          <property name="text">
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="print_btn">
          <property name="text">
           <string>Print</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="front_btn">
          <property name="text">
           <string>Front</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back_btn">
          <property name="text">
           <string>Back</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="executeTask_btn">
          <property name="text">
           <string>Execute Task</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="output_widget"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>877</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
